### ISSUE TITLE
how to use proposed dataset

### ISSUE THREAD

**Issue by duongnv0499** [microsoft/SDR#14]
Hi team, thank you for the great work, I want to use your proposed dataset (wines) for my study but I found no ground truth, all in that file wines.txt is only the title and sections. I want to know how ground truth is arranged in this file.
Thank you team! hope you reply soon!

**Comment by dburian** [comment]
Hi, I am too looking for the ground truth labels and unable to find them. @duongnv0499, have you attempted to contact the authors by email?

**Comment by duongnv0499** [comment]
>  yes, each dataset should have a "gt" file at its folder, example https://github.com/microsoft/SDR/blob/main/data/datasets/video_games/gt

**Comment by duongnv0499** [comment]
> Hi, I am too looking for the ground truth labels and unable to find them. @duongnv0499, have you attempted to contact the authors by email?

yes, each dataset should be a "gt" file in its folder, example : https://github.com/microsoft/SDR/blob/main/data/datasets/video_games/gt

**Comment by dburian** [comment]
Thanks. I don't know how I missed that...
